<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: rgb(255, 255, 255);
border-radius:15px;
}
QToolButton#closeButton,QToolButton#minimizeButton{
color:rgb(0,0,0);
border-style:none;
}
QToolButton#closeButton::hover{
color:rgb(255,255,255);
background-color: rgba(255, 56, 56, 150);
border-top-right-radius:15px;
}
QToolButton#minimizeButton::hover{
border-style:none;
color:rgb(0,0,0);
background-color: rgba(203, 203, 203, 150);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/img/logonbg.ico</pixmap>
    </property>
   </widget>
   <widget class="QToolButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>╳</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QFrame" name="funcFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>60</width>
      <height>560</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#funcFrame{
border:none;
background-color:rgb(46, 46, 46);
border-top:1px solid rgb(116, 116, 116);
border-bottom-left-radius:15px;
}
QToolButton#addFriendBT{
background-color:rgb(46, 46, 46);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QToolButton" name="addFriendBT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>185</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>添加好友</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
	background-color:rgb(46, 46, 46);
    border: none;
   image: url(:/img/addFriends.png)
}

QToolButton:hover {
	background-color:rgb(158, 158, 158);
	border-radius:3px;
}
QToolButton:pressed {
    image: url(:/img/pressed-addFriends.png)  /*点击时的图片 */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="chatListBT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>聊天列表</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
	background-color:rgb(46, 46, 46);
    border: none;
}

QToolButton:hover{
	background-color:rgb(158, 158, 158);
	border-radius:3px;
}

QToolButton:checked {
	background-color:rgb(223, 223, 223);
	border-radius:3px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/chatList.png</normaloff>:/img/chatList.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="friendsListBT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>好友列表</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
	background-color:rgb(46, 46, 46);
    border: none;
}

QToolButton:hover{
	background-color:rgb(158, 158, 158);
	border-radius:3px;
}

QToolButton:checked {
	background-color:rgb(223, 223, 223);
	border-radius:3px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/friendsList.png</normaloff>:/img/friendsList.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="listFrame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>240</width>
      <height>440</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#listFrame{
border:none;
background-color:rgb(229, 228, 228);
border-top:1px solid rgb(116, 116, 116);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="dialogFrame">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>600</width>
      <height>560</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border:none;
	border-top:1px solid rgb(116, 116, 116);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="minimizeButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>一</string>
    </property>
   </widget>
   <widget class="QFrame" name="infoFrame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>240</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#infoFrame{
border:none;
background-color:rgb(233, 248, 255);
border-top:1px solid rgb(116, 116, 116);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QToolButton" name="avatarBT">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    border: none;
	border-radius:3px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/logo.png</normaloff>:/img/logo.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>duangxin</string>
     </property>
    </widget>
    <widget class="QLabel" name="idLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:00712</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
